roles/mariadb/tasks/main.yml

```diff
@@ -17,10 +17,18 @@
     name: mariadb-server
     state: present
 
-- name: Start MariaDB MySQL Server
+- name: Disable MariaDB binary logging
+  when: binary_logging_disabled
```

Comment:
Small thing but can the `when` condition be put after the `template` action? That way it's consistent with the rest of the codebase.